notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>Thank you for submitting your work. Seems great! A few comments below:</p><p> </p><p>Don't quite understand the meaning of the sentence "The case 2 is a household electricity meter, the screen of the meter is LCD, which means we can note only see the text but also the outline of the text in the background. " Do you mean "not only read the text from the LCD screen, but also...? What's the problem of "see the outline of the text in the background?</p><p> </p><p>Also, it seems that LCD screen only needs one more step compared to the 1st type of meters. Could you combine the pipelines of the 2 types of meters, and then adding one more branch for the additional pre-processing that LCD screen requires?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
hi, I add some words ‘The background outline can also be mistaken by the model as content that needs to be recognized, so we need to add some preprocessing to eliminate the impact of background information.’.

Hope the words can solve your confusion!